src/main/webapp/feed.html

```diff
@@ -6,9 +6,44 @@
       onload="loadHeader('Message Feed')"
     ></script>
     <script src="js/feed-page-loader.js"></script>
-    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.9.0/css/all.css">
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.9.0/css/all.css"
+    />
```

Comment:
Saw that fontawesome was added locally, can this be replaced with a reference to the local file instead of fetching from the URL?

Reply:
Ok sure! Did not see this one.